infra/main.tf

```diff
@@ -473,11 +513,60 @@ module "test_gsa_secret" {
   ]
 }
 
+resource "google_storage_bucket_iam_member" "test_bucket_admin" {
+  bucket = module.hail_test_gcs_bucket.name
+  role = "roles/storage.admin"
+  member = "serviceAccount:${module.test_gsa_secret.email}"
+}
+
+resource "google_storage_bucket_iam_member" "test_requester_pays_viewer" {
+  bucket = module.hail_requester_pays_gcs_bucket.name
+  role = "roles/storage.objectViewer"
+  member = "serviceAccount:${module.test_gsa_secret.email}"
+}
+
+resource "google_storage_bucket_iam_member" "test_gcr_viewer" {
+  bucket = google_container_registry.registry.id
+  role = "roles/storage.objectViewer"
+  member = "serviceAccount:${module.test_gsa_secret.email}"
+}
+
 module "test_dev_gsa_secret" {
   source = "./gsa_k8s_secret"
   name = "test-dev"
 }
 
+resource "kubernetes_secret" "hail_ci_0_1_github_oauth_token" {
+  metadata {
+    name = "hail-ci-0-1-github-oauth-token"
+  }
+
+  data = {
+    "oauth-token" = var.ci_github_oauth_token
+  }
+}
+
+resource "kubernetes_secret" "zulip_config" {
+  metadata {
+    name = "zulip-config"
+  }
+
+  data = {
+    ".zuliprc" = file("~/.hail/.zuliprc")
+  }
+}
+
+resource "kubernetes_secret" "auth_oauth2_client_secret" {
```

Comment:
Can we rename this as `google_oauth2_client_secret`?

Reply:
I suppose this might depend on what we hear back about azure oauth, but I imagined that oauth2 being a standard would mean we would have the same secret just filled by an azure oauth2 secret, whatever form that may take. In which case, why make the name of this secret specifically google?